src/hotspot/share/opto/loopnode.cpp

```diff
@@ -3990,7 +3990,6 @@ void IdealLoopTree::dump_head() const {
     if (cl->is_post_loop()) tty->print(" post");
     if (cl->is_reduction_loop()) tty->print(" reduction");
     if (cl->is_vectorized_loop()) tty->print(" vector");
-    if (cl->range_checks_present()) tty->print(" rc ");
```

Comment:
Can you use new `range_checks_present()` here? The information is still useful.

Reply:
Thanks for reviewing this.
Indeed, you're right. I updated the change.